checkbox-ng/checkbox_ng/launcher/subcommands.py

```diff
@@ -314,8 +331,18 @@ def _configure_restart(self, ctx):
             lambda session_id: [shlex.join(respawn_cmd.append(session_id))]
         )
 
-    def _maybe_resume_session(self):
-        resume_candidates = list(self.ctx.sa.get_resumable_sessions())
+    def _maybe_auto_resume_session(self, resume_candidates):
+        """
+        import pdb
+
+        pdb.set_trace()
+        sessions = [session.id for session in resume_candidates]
+        if sessions:
+            selected_session_id = ResumeSessionBrowser(sessions).run()
+            raise SystemExit(selected_session_id)
+        print("no sessions")
+        """
+
```

Comment:
What is this for? Is it a docstring?

Reply:
No, that's me being sloppy in debug/dead code removal :face_with_spiral_eyes: